tests/nb/cover_HassTurnOff.yaml

```diff
@@ -22,18 +22,45 @@ tests:
           - blind
           - curtain
           - shutter
-    response: Lukket kjøkken
+    response: Lukket takvinduet
```

Comment:
I like how {cover_classes:device_class} picks up that we asked for [tak]vinduet, and feeds the same back in the response.